arrays/arrays.c

```diff
@@ -128,7 +129,7 @@ void arr_append(Array *arr, char *element) {
   // Resize the array if the number of elements is over capacity
   // or throw an error if resize isn't implemented yet.
   if (arr->count + 1 > arr->capacity){
-    fprintf(stderr, "IndexError: Array at capacity");
+    resize_array(arr);
     return;
```

Comment:
With this return here, the function will not continue to copy the element and add it to the array. The function will stop after resize_array(arr).